.github/workflows/trybot.yml

```diff
@@ -1,61 +1,53 @@
-on:
+"on":
   push:
     branches:
       - master
-  repository_dispatch:
+  repository_dispatch: {}
   pull_request:
     branches:
       - '**'
-
 name: Test
 jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v3
-    - name: Install Node
-      uses: actions/setup-node@v3
-      with:
-        node-version: '18.9.0'
-    - name: Install Go
-      uses: actions/setup-go@v3
-      with:
-        go-version: '1.19.1'
-    - name: Ensure latest CUE
-      # The latest git clean check ensures that this call is effectively
-      # side effect-free. Using GOPROXY=direct ensures we don't accidentally
-      # hit a stale cache in the proxy.
```

Comment:
are you losing comments on purpose?

Reply:
I guess you are, thinking about it again, since we'll have the comments in the CUE source of truth.

Reply:
Yep, exactly.